web/cltvspj/views.py

```diff
@@ -2,7 +2,81 @@
 
 # Create your views here.
 def index(request):
-    return render(request, 'cltvspj/form.html')
+
+    melhor = {}
+    melhor['salario'] = ''
+
+    if(request.method == 'GET'):
+       return render(request, 'cltvspj/form.html', {'melhor': melhor})
+
+    elif(request.method == 'POST'):
+        salario_pj =  request.POST['PJ']
+        salario_clt = request.POST['CLT']
+        # iss = request.POST['ISS']
+        # irrf = request.POST['IRRF']
+
+        RBT_ANO = request.POST['RBT_ANO']   # Receita bruta anual
```

Comment:
change to receita_bruta_anual = request.POST['receita_bruta_anual], don't simplify variable names